scripts/start.py

```diff
@@ -52,6 +52,10 @@
     '--enable_console',
     help='optional; if specified, enables console.',
     action='store_true')
+_PARSER.add_argument(
```

Comment:
The changes here are unrelated to the PR, but this would be quite helpful to quickly enable access to the server across multiple devices on the same  network. Should I create a separate PR for this, or is this fine here?

Reply:
Removed it from this PR, will create another one for this.